app/presenters/monitoring-stations/base.presenter.js

```diff
@@ -0,0 +1,129 @@
+'use strict'
+
+const { formatLongDate, sentenceCase, formatAbstractionPeriod } = require('../base.presenter.js')
+
+function _alert(status, statusUpdatedAt) {
+  if (!statusUpdatedAt) {
+    return null
+  }
+
+  return sentenceCase(status)
+}
+
+/**
+ * Formats a list of licence monitoring station records into abstraction restrictions.
+ *
+ * Each input object may include an `action` property with the following structure:
+ * ```
+ * action = {
+ *   link: `/system/licence-monitoring-station/${licenceMonitoringStation.id}`,
+ *   text: 'View'
+ * }
+ * ```
+ *
+ * This action is passed through to the resulting restriction object unchanged.
+ *
+ * @param {object[]} licenceMonitoringStations
+ *
+ * @returns {object[]}
+ */
+function restrictions(licenceMonitoringStations) {
```

Comment:
What do you know!? @Cruikshanks breaks his own rules!

If you look at all our other base presenters, the helper functions _do_ have something in front of what they return that gives an idea of what they are doing with the data.

In this case, you've said it in the function description.

```suggestion
function formatRestrictions(licenceMonitoringStations) {
```